scalameta/trees/shared/src/main/scala/scala/meta/Trees.scala

```diff
@@ -438,6 +439,8 @@ object Ctor {
     self: Self,
     stats: List[Stat]
 ) extends Tree {
+  @binaryCompatField
+  private var derives: List[Init] = Nil
```

Comment:
Can we name the private field `_derives` and the public field `derives`? It's inconsistent to use `.getDerives` for this field and `stats` for other fields.

Reply:
Changed.

Reply:
Setter is still called setDerives though.